### PR TITLE
SEO-176062-Windowsforms-UG-duplicate-title

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Have updated UG titles please review.
Thanks.

Regards,
Edith Abich.

### CODE REVIEW

File: WindowsForms/GridRecordNavigationControl/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms GridRecordNavigationControl | Syncfusion
+title: Overview of Windows Forms GridRecordNavigationControl | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/GridRecordNavigationControl/Overview.md:3]: Overview of Windows Forms Grid Record Navigation Control | Syncfusion

File: WindowsForms/Numeric-TextBox/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms Numeric TextBox control | Syncfusion
+title: Overview of Windows Forms Numeric TextBox control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Numeric-TextBox/Overview.md:3]: Control the first word should be in caps, check in all places

File: WindowsForms/Scroll-Frame/OverView.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms Scroll Frame control | Syncfusion
+title: Overview of Windows Forms Scroll Frame control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Scroll-Frame/OverView.md:3]: Control the first word should be in caps.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Scroll-Frame/OverView.md (hunk 3 above)
Comment (Edithabich) [reply WindowsForms/Scroll-Frame/OverView.md:3]: Resolved

File: WindowsForms/Numeric-TextBox/Overview.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/Numeric-TextBox/Overview.md:3]: Resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request closed without merging**